packages/docs/operations-manual/key-management/detailed.md

```diff
@@ -0,0 +1,148 @@
+# Detailed Description of Key Types
+This page provides a detailed description of the various key prototypes found in the Celo protocol, as well as small examples of how to designate key types in the Celo proof-of-stake smart contracts.
+
+## Undesignated
+
+By default, any private key generated in the Celo protocol does not have a specific role in proof-of-stake. These keys can be used to sign and send transactions that are unrelated to Celo's proof-of-stake protocol, including but by no means limited to transferring Celo Gold, transferring Celo Dollars, exchanging Celo Gold for Celo Dollars, etc.
+
+Undesignated keys can be designated as Locked Gold Accounts or authorized as signer keys on behalf of a Locked Gold Account by sending special transactions using `celocli`.
+
+Note that [ReleaseGold](../../celo-codebase/protocol/release-gold) beneficiary keys are by default Undesignated with respect to proof-of-stake, and that the `ReleaseGold` contract address is what ultimately gets designated as a Locked Gold Account.
+
+
+## Locked Gold Accounts
+
+Locked Gold Account keys have the highest level of privileges in the Celo protocol. These keys can be used to lock and unlock cGLD in order to be used in proof-of-stake. Furthermore, Locked Gold Account keys can be used to authorize other keys to sign transactions and messages on behalf of the Locked Gold Account.
```

Comment:
I would just use Account everywhere and not use keys since it is implied that you need a private key for an account

Reply:
Yeah I think your suggestion around "celo accounts" lead towards a much clearer and consistent terminology. Only using "key" from here on out when referencing the private key material directly (e.g. keep your key on a ledger)